tests/ops/integration_tests/saas/test_mailchimp_transactional_task.py

```diff
@@ -0,0 +1,214 @@
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+
+from fides.api.ops.models.policy import ActionType
+from fides.api.ops.models.privacy_request import (
+    ExecutionLog,
+    ExecutionLogStatus,
+    PrivacyRequest,
+)
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.schemas.saas.shared_schemas import SaaSRequestParams
+from fides.api.ops.service.connectors import get_connector
+from fides.api.ops.service.privacy_request.request_runner_service import (
+    build_consent_dataset_graph,
+)
+from fides.api.ops.task import graph_task
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mailchimp_transactional
+def test_mailchimp_transactional_connection_test(
+    mailchimp_transactional_connection_config,
+) -> None:
+    get_connector(mailchimp_transactional_connection_config).test_connection()
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mailchimp_transactional
+@pytest.mark.asyncio
+async def test_mailchimp_transactional_consent_request_task(
+    db,
+    consent_policy,
+    mailchimp_transactional_connection_config,
+    mailchimp_transactional_dataset_config,
+    mailchimp_transactional_identity_email,
+) -> None:
+    """Full consent request based on the Mailchimp Transactional (Mandrill) SaaS config"""
+
+    privacy_request = PrivacyRequest(
+        id=str(uuid4()),
+        consent_preferences=[{"data_use": "advertising", "opt_in": False}],
+    )
+
+    identity = Identity(**{"email": mailchimp_transactional_identity_email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = "mailchimp_transactional_instance"
+
+    v = await graph_task.run_consent_request(
+        privacy_request,
+        consent_policy,
+        build_consent_dataset_graph([mailchimp_transactional_dataset_config]),
+        [mailchimp_transactional_connection_config],
+        {"email": mailchimp_transactional_identity_email},
+        db,
+    )
+
+    assert v == {
+        f"{dataset_name}:{dataset_name}": True
+    }, "graph has one node, and request completed successfully"
+
+    execution_logs = db.query(ExecutionLog).filter_by(
+        privacy_request_id=privacy_request.id
+    )
+
+    mailchimp_transactional_logs = execution_logs.filter_by(
+        collection_name=dataset_name
+    ).order_by("created_at")
+    assert mailchimp_transactional_logs.count() == 2
+
+    assert [log.status for log in mailchimp_transactional_logs] == [
+        ExecutionLogStatus.in_processing,
+        ExecutionLogStatus.complete,
+    ]
+
+    for log in mailchimp_transactional_logs:
+        assert log.dataset_name == dataset_name
+        assert (
+            log.collection_name == dataset_name
+        ), "Node-level is given the same name as the dataset name"
+        assert log.action_type == ActionType.consent
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mailchimp_transactional
+@pytest.mark.asyncio
+async def test_mailchimp_transactional_consent_request_task(
```

Comment:
is this a dupe of the method above? at a minimum, seems like the method signature is duplicated, which surely isn't intentional?

or maybe i'm going crazy :)

Reply:
thanks for catching, I just need to remove!